pkgs/servers/flaresolverr/default.nix

```diff
@@ -0,0 +1,74 @@
+{ pkgs, stdenv }:
+with pkgs;
+
+stdenv.mkDerivation rec {
+
+  pname = "flaresolverr";
+  version = "2.2.6";
+
+  src = fetchurl {
+    url = "https://github.com/FlareSolverr/FlareSolverr/releases/download/v${version}/flaresolverr-v${version}-linux-x64.zip";
```

Comment:
Is there a reason to use the prebuilt version? This excludes ARM and darwin.

Reply:
You are right, I forgot about this problem. I'll close for now and come back with a version packaged from sources.